support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -201,15 +202,17 @@ const CardForm = (props: PropTypes) => {
     }
 
     window.grecaptcha.render('robot_checkbox', {
-      sitekey: window.guardian.v2recaptchaPublicKey,
+      sitekey: props.postDeploymentTestUser ?
```

Comment:
postDeploymentTestUser rather than isTestUser?
why are we passing isTestUser in if it's not used?

Reply:
if it is done off post deploy user, we shouldn't really call it uat/default, as those are test user terms.

Reply:
I've refactored this whole area now, it didn't need to know about post deploy users, test user will do.